lib/membrane_rtc_engine/endpoints/webrtc/track_sender.ex

```diff
@@ -68,11 +85,35 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.TrackSender do
         _other -> []
       end)
 
+    actions = actions ++ [start_timer: {:check_encoding_statuses, Time.seconds(1)}]
```

Comment:
Shouldn't be this time somehow configurable?

Reply:
It hasn't been so far so I would move this to a separate issue